workspaces/blackduck/examples/entities.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
   annotations:
     blackduck/project: example-host/example-website/main
 spec:
-  type: website
+  type: service
```

Comment:
This is called `example-website` would it not make more sense to create an `example-service` instead and leave this as is?

Reply:
Oh, definitely, good point, I somehow missed it, let me change it 😄